profiler/src/ProfilerEngine/Datadog.Profiler.Native/LibddprofExporter.cpp

```diff
@@ -18,6 +18,10 @@
 #include <string.h>
 #include <time.h>
 
+#ifndef _WINDOWS
+#include <filesystem>
+#endif
```

Comment:
not needed
this is included by the `dd_filesystem.hpp` line 25

Reply:
https://github.com/DataDog/dd-trace-dotnet/blob/master/shared/src/native-src/dd_filesystem.hpp
and you can use `fs::` for the namespace